templates/docker-compose.yaml.eta

```diff
@@ -0,0 +1,21 @@
+services:
+  <%= it.projectName %>:
+    restart: always
+    environment:
+      - WATCHPACK_POLLING=true
```

Comment:
This is for Windows users, but I am not sure if it works because I don't have a Windows machine.